drivers/display/ssd1306.c

```diff
@@ -458,7 +458,8 @@ static int ssd1306_init(struct device *dev)
 
 	gpio_pin_configure(driver->reset,
 			   DT_INST_0_SOLOMON_SSD1306FB_RESET_GPIOS_PIN,
-			   GPIO_DIR_OUT);
+			   GPIO_OUTPUT_HIGH |
+			   DT_INST_0_SOLOMON_SSD1306FB_RESET_GPIOS_FLAGS);
```

Comment:
I assume that polarity will be passed by `DT_INST_0_SOLOMON_SSD1306FB_RESET_GPIOS_FLAGS`?

Reply:
Yes, I did not fixed it in shield overlays because gpio bruch is outdated and has conflicts with master.